examples/pylab_examples/quiver_demo.py

```diff
@@ -72,7 +55,6 @@
 plt.axis([-1, 7, -1, 7])
 plt.title("triangular head; scale with x view; black edges")
 
-# 6
 plt.figure()
```

Comment:
agree with @NelleV that there are a lot of examples here; it's kinda overwhelming and is there a reasonable way for it to be pared down? Specially since this is a pylab example and so all these features are (or should be) documented the OO way?